src/convert.js

```diff
@@ -163,8 +163,22 @@ function bytes2str (buf) {
  * @param {string | Uint8Array | CID} hash
  */
 function mh2bytes (hash) {
+  let cid
+
+  if (typeof hash === 'string') {
```

Comment:
We have this same logic in `PeerId` as well. Thinking if cid could offer a way of doing it?

Reply:
Could probably add it to `CID.asCID`, though I think that's supposed to convert between old and new CID instances, not try to parse whatever we throw at it.  This is quite a common use case though.

Reply:
>  This is quite a common use case though

Can we export an util static function for this in multiformats then?

Reply:
I've created https://github.com/multiformats/js-multiformats/issues/92